src/openrct2/actions/GameAction.cpp

```diff
@@ -66,6 +95,77 @@ namespace GameActions
         return false;
     }
 
+    void Enqueue(const GameAction* ga, uint32_t tick)
```

Comment:
Any reason not to declare as `const GameAction& ga`?

Reply:
Only for consistency reasons, Query/Execute also take the same parameter, should make them maybe references in another PR.